korangar/src/loaders/animation/mod.rs

```diff
@@ -258,14 +266,50 @@ impl AnimationLoader {
 
                 return Vector2::new(size_x, size_y);
             }
+            #[cfg(feature = "debug")]

```

Comment:
Please add a free line before this. Also: Do we need thos function to be inlined? Can't they be a free standing function?

Reply:
They can be free standing, just not sure where to put the function.